types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -3,10 +3,13 @@ import {ChannelList} from '../../database/schema/ChannelList.types';
 interface UseChatUtilsHook {
   selectedChannel: ChannelList | null;
   goBack: () => void;
-  goToChatScreen: (channel: ChannelList) => void;
+  goToChatScreen: (channel: ChannelList, from?: string) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `goToChatScreen` function now accepts an optional second parameter `from`. This change could potentially affect all the places where this function is being called. Make sure to update the function calls accordingly in other parts of the codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->